src/components/work/recommendations/Recommendations.js

```diff
@@ -53,7 +57,8 @@ export default function Recommendations({ workId }) {
     <Section
       title={Translate({ ...context, label: "remindsOf" })}
       dataCy="section-recommend"
-      divider={{ content: false }}
+      divider={{ content: false, title: titleDivider }}
+      headerTag={!!headerTag ? headerTag : undefined}
```

Comment:
Bare af interesse, hvorfor "undefined" i stedet for "null"?

Reply:
optional props ville være undefined hvis de ikke er sat. her ønsker jeg ikke at sætte den

Reply:
hmm har lige indset at det er det samme som bare at sætte headerTag her..